listener.go

```diff
@@ -366,6 +364,15 @@ func (l *listener) loadCert() (*tls.Certificate, error) {
 		return nil, err
 	}
 
+	// cert has changed, close closeWrapper wrapped connections
+	if l.conns != nil {
+		l.connLock.Lock()
+		for _, conn := range l.conns {
+			_ = conn.close()
+		}
+		l.connLock.Unlock()
+	}
+
```

Comment:
So, prior to this change, connections were reset when calling `updateCert()` but with this change it is on `loadCert()` which leads to different behavior when calling `getCertificate()`. Is this intentional?

Reply:
There was a defect with the connection reset code, where only the node updating the certificate would reset its connections. The other nodes would load the new cert from the k8s store, but not close any connections, leaving them on the old cert. This resolves that.